docs/data/material/components/snackbars/snackbars.md

```diff
@@ -40,6 +38,26 @@ You can control the position of the snackbar by specifying the `anchorOrigin` pr
 
 {{"demo": "PositionedSnackbar.js"}}
 
+## Stacked snackbars
+
+The `Snackbar` component doesn't yet support displaying multiple stacked snackbars out of the box.
+However, there are currently two complementary alternatives to do that:
+
+### Lab's stacked snackbars
+
+We have been developing a native component to support this use case and [we've made it available in the lab](/material-ui/react-stacked-snackbars/) to gather feedback for its first versions and general API design.
+
+### notistack
+
+![stars](https://img.shields.io/github/stars/iamhosseindhv/notistack.svg?style=social&label=Stars)
+![npm downloads](https://img.shields.io/npm/dm/notistack.svg)
+
+This example demonstrates how to use [notistack](https://github.com/iamhosseindhv/notistack).
+notistack has an **imperative API** that makes it easy to display snackbars, without having to handle their open/close state.
+It also enables you to **stack** them on top of one another (although this is discouraged by the Material Design guidelines).
+
+{{"demo": "IntegrationNotistack.js", "defaultCodeOpen": false}}
```

Comment:
It sounds like we can replace all mentions of notistack in the docs now

```suggestion
```

Reply:
@danilo-leal Can you take a look?

Reply:
Addressed in https://github.com/mui/material-ui/pull/31991/commits/7a351f66c7efecb43be56c34a782349da7494594  👍